packages/frontend/src/components/claims/index.tsx

```diff
@@ -1,12 +1,76 @@
 "use client";
 
 import { useClaims } from "@/src/hooks/useClaims";
+import { Chains } from "./chains";
+import { SupportedChain, type Claim } from "@metrom-xyz/sdk";
+import { useMemo, useState } from "react";
+import { type Chain } from "viem";
+import { celoAlfajores, holesky, mantleSepoliaTestnet } from "viem/chains";
+import { ChainOverview } from "./chain-overview";
 
+import styles from "./styles.module.css";
+
+export interface ChainWithClaimsData {
+    chain: Chain;
+    claims: Claim[];
+}
+
+// TODO: i18n
 export function Claims() {
     const { loading, claims } = useClaims();
 
-    // TODO: remove this
-    console.log({ loading, claims });
+    const chainsWithClaimsData = useMemo(() => {
+        if (loading || claims.length === 0) return [];
+        const reduced = claims.reduce(
+            (acc, claim) => {
+                let data = acc[claim.chainId as SupportedChain];
```

Comment:
`data` can be a const here, no?

Reply:
Addressed in fc2090d0f245c01946f822083b8adbd743cd8f78